ubuntu-deploy/index.md

```diff
@@ -0,0 +1,414 @@
+
+Let's step through the service file:
+
+- [Unit] Section:
+
+  - Description: Description of the systemd service.
+  - Requires: Dependency on the Docker service.
+  - After: Starts after the Docker service.
+
+- [Service] Section:
+
+  - Restart=always: The service restarts automatically on exit.
+  - WorkingDirectory: Path to the directory containing the docker-compose.yml file.
+  - ExecStart: Command to start Docker containers.
+  - ExecStop: Command to stop Docker containers.
+
+- [Install] Section:
+  - WantedBy=multi-user.target: Service enabled on reaching the multi-user state.
+
+### Enable and Start the Service
+
+Enable the service to start on boot, and then start the service:
+
+```bash
+sudo systemctl enable directus.service
+sudo systemctl start directus.service
+```
+
+Now, your dockerized Directus application is running as a background service and will automatically restart in case of failures or system reboots.
+
+Run the following command to check the status of the service:
+
+```bash
+sudo systemctl status directus.service
+```
+
+![Directus service status][image-7]
+
+You can also confirm if your application is still running at `http://your_server_ip:8055`.
+
+## Configuring DNS Settings for Your Domain
+
+Configuring DNS settings for your domain is a crucial step in making your Directus application accessible to users over the internet. Here's how to do it:
+
+1. Access Your Domain Registrar's Website: Log in to the website of your domain registrar, where you initially purchased or registered your domain name. This is where you manage your domain settings.
+
+2. Locate DNS Management or Domain Settings: Inside your domain registrar's dashboard, look for options like "DNS Management," "Domain Settings," or "Domain Management." These names might vary based on the registrar's interface.
+
+![DNS management dashboard][image-8]
+
+3. Add a DNS Record for Your Subdomain: Create a new DNS record to point your subdomain (e.g., directus.exampledomain.com) to your server's public IP address. Depending on the registrar, you may need to choose the record type, which is usually an "A" record for IPv4 addresses. Enter your server's public IP address in the designated field.
+
+![A record domain configuration][image-5]
+
+4. Save the changes: After adding the DNS record, save the changes. DNS propagation might take some time, ranging from a few minutes to a few hours. During this period, the DNS changes will propagate across the internet, making your subdomain accessible.
+
+You can confirm your changes by visiting the application by visiting `http://directus.exampledomain.com:8055`.
+
+![Directus application accessed by the domain at port 8055][image-4]
+
+## Setting Up Nginx as a Reverse Proxy
+
+Nginx, often called 'engine-x,' is a powerful reverse proxy server widely used in web hosting.
+
+As a reverse proxy, Nginx sits between clients and backend servers, forwarding client requests to the appropriate server.
```

Comment:
Can we add another line or two explaining the 'why' of Nginx? I also think we can remove the pronunciation guide as it's unimportant and we haven't done it elsewhere.